.dockerignore

```diff
@@ -1 +1,2 @@
 .git
+.venv
```

Comment:
If you have a uv virtualenv locally, then this prevents it from ending up in the Docker image from the `COPY` which then causes other problems.